integration/overrides_exporter_test.go

```diff
@@ -1,119 +0,0 @@
-// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
Why was it removed? :)

Reply:
I lost it somewhere along the way, did not intend to remove it for good^^